tests/ui/inline_format_args.rs

```diff
@@ -0,0 +1,108 @@
+// run-rustfix
+
+#![allow(clippy::eq_op)]
+#![allow(clippy::format_in_format_args)]
+#![allow(clippy::print_literal)]
+#![allow(named_arguments_used_positionally)]
+#![allow(unused_variables)]
+#![warn(clippy::inline_format_args)]
+
+fn tester(fn_arg: i32) {
+    let local_value = 1;
+    let local_value2 = 2.0;
+    let local_opt: Option<i32> = Some(3);
+    let width = 4;
+    let prec = 5;
+    let val = 6;
+
+    // make sure this file hasn't been corrupted with tabs converted to spaces
+    assert_eq!("	 	", "\t \t");
+
+    println!("val='{}'", local_value);
+    println!("val='{   }'", local_value);
+    println!("val='{	}'", local_value);
+    println!("val='{ 	}'", local_value);
+    println!("val='{	 }'", local_value);
+    println!("val='{\t }'", local_value);
+    println!("val='{\n }'", local_value);
+    println!(
+        "val='{
+    }'",
+        local_value
+    );
+    println!("val='{local_value}'", local_value = local_value);
+    println!("val='{local_value}'", local_value = fn_arg);
+    println!("{}", local_value);
+    println!("{}", fn_arg);
+    println!("{:?}", local_value);
+    println!("{:#?}", local_value);
+    println!("{:4}", local_value);
+    println!("{:04}", local_value);
+    println!("{:<3}", local_value);
+    println!("{:#010x}", local_value);
+    println!("{:.1}", local_value2);
+    println!("Hello {} is {:.*}", "x", local_value, local_value2);
+    println!("{0}", local_value);
```

Comment:
@flip1995 I could, but with the recent changes to the underlying format string parsing code, it might be possible for me to handle more cases - which SHOULD be inlined, but I didn't have the needed info to handle them.